db/migrate/20160920144053_change_profile_to_json.rb

```diff
@@ -0,0 +1,45 @@
+class ChangeProfileToJson < ActiveRecord::Migration
+  class Profile < ActiveRecord::Base
+  end
+
+  def up
+    add_column :profiles, :encrypted_pii, :text
+    Profile.all.each do |profile|
+      pii = {
+        first_name: profile.first_name,
+        middle_name: profile.middle_name,
+        last_name: profile.last_name,
+        address1: profile.address1,
+        address2: profile.address2,
+        city: profile.city,
+        state: profile.state,
+        zipcode: profile.zipcode,
+        ssn: profile.ssn,
+        dob: profile.dob.to_s,
+        phone: profile.phone
+      }
+      profile.update!(encrypted_pii: pii.to_json, active: false)
```

Comment:
Just to make sure -- defining `Profile` in here will ensure that we don't run into problems with failing AR validations later on, right?  (if, for instance, this migration is run at a time when there are new AR validations but that column hasn't actually been added to the DB yet, so AR data updates will fail)